localstack/utils/bootstrap.py

```diff
@@ -249,9 +281,8 @@ def get_enabled_apis() -> Set[str]:
     return resolve_apis(services)
 
 
-# DEPRECATED, lazy loading should be assumed
-def is_api_enabled(api: str) -> bool:
-    apis = get_enabled_apis()
+def should_eager_load_api(api: str) -> bool:
+    apis = get_preloaded_services()
 
     if api in apis:
         return True
```

Comment:
nit:
```suggestion
def should_eager_load_api(api: str) -> bool:
    return api in get_preloaded_services()
```

Reply:
Then we will remove the check for the `port` part that we had previously. I'm okay with removing it, but we had specific tests in place to remove the port part like `SERVICES=sqs:123` However, those would not work with `STRICT_SERVICE_LOADING`, so I will remove.

Reply:
I'm a bit confused, this change request is semantically equivalent, right?

Reply:
It's missing a part:
```python
def should_eager_load_api(api: str) -> bool:
    apis = get_preloaded_services()
    if api in apis:
        return True
    for enabled_api in apis:
        if api.startswith(f"{enabled_api}:"):
            return True
    return False
```
But it's good we're removing this, this is very old and will be good to be removed

Reply:
Oh, that was just due to the review view in GitHub. I haven't seen the lines below. Yours to judge if it's time to remove this. Otherwise, please ignore the comment!